utils/resources.go

```diff
@@ -207,6 +207,17 @@ var resourceRequirements = map[string]corev1.ResourceRequirements{
 			"cpu":    resource.MustParse("20m"),
 		},
 	},
+
+	"kube-rbac-proxy": {
+		Limits: corev1.ResourceList{
+			"memory": resource.MustParse("30Mi"),
+			"cpu":    resource.MustParse("50m"),
+		},
+		Requests: corev1.ResourceList{
+			"memory": resource.MustParse("30Mi"),
+			"cpu":    resource.MustParse("50m"),
+		},
+	},
```

Comment:
@bindrad How did you choose these values?

Reply:
Similar to kube-rbac-proxy container for deployer pod